stories/form/form-textarea/form-textarea.stories.mdx

```diff
@@ -83,6 +77,23 @@ import { Textarea } from "@lifesg/react-design-system/input-textarea";
     </Story>
 </Preview>
 
+<Heading3>Textarea with custom counter label</Heading3>
+
+<Preview>
+    <Story name="With custom counter">
+        <StoryContainer>
+            <Container>
+                <Form.Textarea
+                    label="This is a textarea with a custom counter label"
+                    placeholder="Enter here..."
+                    maxLength={100}
+                    renderCustomCounter={renderCustomCounter}
+                />
+            </Container>
+        </StoryContainer>
+    </Story>
+</Preview>
+
```

Comment:
Add a simple code snippet for readers to understand what is inside the `renderCustomCounter`. You can do something like

```
```tsx
const renderCustomCounter = (maxLength: number, currentValueLength: number) => {
      return <CustomLabel>{currentValueLength} / {maxLength}</CustomLabel>;
};
```
```